modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -0,0 +1,9 @@
+[id="creating-a-new-client-in-keycloak_{context}"]
+= Creating a new client in Keycloak
+
+Create a new client in Keycloak for secure and easy identity and access management. Keycloak acts as an Identity broker between ArgoCD and OpenShift, as a result you can login into Argo CD using the OpenShift credentials also.
+
+.Procedure
+
+. Configure Keycloak by editing the Argo CD and updating `.sso.spec.provider: keycloak`.
```

Comment:
I read this step this way,

Configure Keycloak by editing **and** updating the Argo CD  **with** `.sso.spec.provider: keycloak`.

How it's currently written makes me think you configure Keycloak by editing the Argo CD and you edit the Argo CD by updating it with this sso spec. Is that right?